readyapi4j-core/src/main/java/com/smartbear/readyapi4j/assertions/InvalidHttpStatusCodesAssertionBuilder.java

```diff
@@ -1,12 +1,65 @@
 package com.smartbear.readyapi4j.assertions;
 
+import com.smartbear.readyapi.client.model.Assertion;
 import com.smartbear.readyapi.client.model.InvalidHttpStatusCodesAssertion;
 import com.smartbear.readyapi4j.AssertionNames;
 import com.smartbear.readyapi4j.Validator;
 
-public class InvalidHttpStatusCodesAssertionBuilder extends ValidHttpStatusCodesAssertionBuilder<InvalidHttpStatusCodesAssertion> {
+import java.util.ArrayList;
+import java.util.List;
 
-    public InvalidHttpStatusCodesAssertionBuilder() {
+public class InvalidHttpStatusCodesAssertionBuilder<T extends Assertion> implements HttpStatusCodeAssertionBuilder {
```

Comment:
I don't see generics (T) being used anywhere in the class. Do we really need it here?